flink-ecosystem-backend/src/main/java/org/apache/flink/community/Application.java

```diff
@@ -50,7 +46,7 @@
     @Autowired
     OAuth2ClientContext oauth2ClientContext;
 
-    @RequestMapping({ "/user", "/me" })
+    @RequestMapping({ "api/v1/user" })
```

Comment:
The behavior of this endpoint is not as documented.
It redirects if the user is not authenticated. 
I would prefer a 405 http status code (https://github.com/sorahn/flink-ecosystem/blob/master/api-proposal.md)

Reply:
will fix it.

Reply:
Here I'll find a way to return a 405 http status, will fix the others and come back to this.

Reply:
There seem to be ways to return a custom HTTP code: https://stackoverflow.com/questions/16232833/how-to-respond-with-http-400-error-in-a-spring-mvc-responsebody-method-returnin